### DISCUSSION TITLE
Sender name in MAILER_EMAIL

### DISCUSSION THREAD

**Discussion by wheeleran** [plausible/analytics#2925] [Anything Else]
This was an unanswered feature request in 2021 (#1443) and a discussion question in 2022 (#2349), and there is still no current way to add a custom Sender name - a MAILER_NAME? - to MAILER_EMAIL. I've tried all versions of the obvious ```"Plausible Mail" <user@example.com>``` but it always returns an error 500. And because there isn't a Name field, email clients such as Gmail fill in the blank with 'user' from 'user@example.com.' 

The point of reports and recipients is surely to be able to share reports with clients, colleagues, and business partners, so being able to actually **name** the Sender is important. 'fluffy-plaus' from 'fluffy-plaus@example.com' doesn't really do it. 

Can we _please_ have this feature?

**Comment by ruslandoga** [comment]
`MAILER_NAME` has been added in in [v2.0.0-rc.1](https://github.com/plausible/analytics/discussions/2981) (https://github.com/plausible/analytics/pull/2937)

**Reply by wheeleran** [reply]
This is great work, thank you so much @ruslandoga
